retirement_api/models.py

```diff
@@ -19,15 +19,21 @@ def __unicode__(self):
 
 class Step(models.Model):
     title = models.CharField(max_length=500)
-    instructions = models.TextField(max_length=255, blank=True)
-    note = models.TextField(max_length=255, blank=True)
+    instructions = models.TextField(blank=True)
+    instructions_es = models.TextField(blank=True)
 
     def __unicode__(self):
         return self.title
 
+    def trans_instructions(self, language='en'):
+        if language == 'es':
+            return self.instructions_es
+        else:
+            return self.instructions
+
     def translist(self):
```

Comment:
Also nitpicky, but would `trans_list` be more within the convention you've been otherwise following?

Reply:
Sorry, didn't look at what this function actually does before commenting.  It's probably out of scope for this PR but it might be nice to rename this regardless or make more explicit how it's being used.